tests/cypress/support/widgets/JupyterLiteSession.ts

```diff
@@ -0,0 +1,129 @@
+import Widget from "./Widget";
+// TODO: celanup eveyrthing
+const selectors = {
+    wrapper: "iframe#jupyter-lite-iframe",
+    main: "#main",
+    notebook: ".jp-Notebook",
+    cellIn: `.jp-Cell-inputWrapper .jp-InputArea-editor`,
+    menuItem: 'li[role="menuitem"]',
+    runTab: 'li[role="menuitem"] > div:contains("Run")',
+    runAllCells: 'li[role="menuitem"] > div:contains("Run All Cells")',
+};
+
+export default class JupyterLiteSession extends Widget {
+    wrappedSelectors: typeof selectors;
+
+    constructor() {
+        super(selectors.wrapper);
+        this.wrappedSelectors = this.getWrappedSelectors(selectors);
+    }
+
+    waitForVisible() {
+        return cy
```

Comment:
It should something like `this.browser.iframe(selectors.wrapper).find(selectors.main)`
Check web-app for examples

Reply:
The problem with that is basically we would need to reimplement all of the methods available in browser for iframeBrowser. Don't quite understand where's the advantage over using `cy` commands that are already fully available and well-documented. 
This iframeBrowser has `#body` as a base for commands, while during chaining I need to apply method to the result, not the`#body`.